rfc/002-extension-content-configuration-processing.md

```diff
@@ -27,6 +27,8 @@ The openMFP Portal dynamically evaluates the installed extensions and surfaces t
 
 The Extension Manager will provide a custom resource definition - Content Configuration - that allows extension providers to describe how their extension should be integrated into the portal in a declarative way. The Extension Manager will then process the CC and store the observed state in its database. The Portal, when rendering the UI will then query for these content configuration resources and consider the processed information from the status.
 
+There is 1-1 relationship between *ExtensionClass* and *ContentConfiguration* resources. To create this relationship a label *extensionClassName* must be configured in the ContentConfiguration containing the name of the associated ExtensionClass.
```

Comment:
Hey @akafazov, shouldn't this be a 1-n relationship where `ExtensionClass` can have multiple `ContentConfiguration`s?

Reply:
changed

Reply:
Nice 👍